src/gui/transferlistwidget.cpp

```diff
@@ -363,14 +363,32 @@ void TransferListWidget::setSelectedTorrentsLocation()
 
 void TransferListWidget::pauseAllTorrents()
 {
+    // Show confirmation if user would really like to Pause All
+    QMessageBox::StandardButton ret =
+            QMessageBox::question(this, tr("Confirm pause"),
+                                  tr("Would you like to pause all torrents?"),
+                                  QMessageBox::Yes | QMessageBox::No, QMessageBox::Yes);
```

Comment:
Also can you try without specifying the last parameter `QMessageBox::Yes`? 
If the dialog defaults to yes button then please omit that parameter.

Reply:
I checked and the behavior is the same, **Yes** is default. **But!** This could be a great idea: I can put "No" by default and then prevent even more mistakes maybe?

I know some will curse me under their breath while trying to auto-yes it, but hey.

(Also N/Y keys work without pressing ALT.)

Reply:
>This could be a great idea: I can put "No" by default and then prevent even more mistakes maybe?

Some users dislike that idea when it was applied to "Confirm delete torrent" dialog and that dialog is even more critical about loosing data than the one here. So I wouldn't make life harder for them for some non-critical dialog.